crates/sui-json-rpc/src/gateway_api.rs

```diff
@@ -166,6 +166,13 @@ impl RpcReadApiServer for GatewayReadApiImpl {
     ) -> RpcResult<Vec<TransactionDigest>> {
         Ok(self.client.get_transactions_in_range(start, end)?)
     }
+
+    async fn get_transaction_auth_signers(
+        &self,
+        _digest: TransactionDigest,
+    ) -> RpcResult<SuiTransactionAuthSignersResponse> {
+        todo!()
```

Comment:
@666lcz I believe the gateway api is being deprecated? unfortunately i had to add here to confirm the trait for read api also